OpenRA.Mods.Common/CampaignProgress.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using OpenRA.FileSystem;
+using OpenRA.Graphics;
+
+namespace OpenRA.Mods.Common
+{
+	public class CampaignProgress
+	{
+		private static string progressFile = Platform.ResolvePath("^", "cnc-progress.yaml");
+		private static bool saveProgressFlag = false;
+        private static string playedMission = "";
```

Comment:
Also in this file. You can leave out the explicit private, too. Have a look at https://github.com/OpenRA/OpenRA/wiki/Coding-Standard